### PR TITLE
OSDOCS-1885: Version number bumps for 4.8 GA

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-1885

### CODE REVIEW

**Comment by jboxman** [comment]
@ahardin-rh, LGTM, but I wonder if we need QE for container images, although if they don't exist that's probably a bigger OCP issue.

File: modules/olm-building-operator-catalog-image.adoc [hunk 1]
```diff
@@ -3,10 +3,10 @@
 // * operators/admin/olm-managing-custom-catalogs.adoc
 
 ifdef::openshift-origin[]
-:registry-image: quay.io/openshift/origin-operator-registry:4.7.0
```
Comment (adellape) [inline modules/olm-building-operator-catalog-image.adoc:6]: This whole module is actually being removed for 4.8 per https://github.com/openshift/openshift-docs/pull/33720, so the changes in the PR here for `modules/olm-building-operator-catalog-image.adoc` can be dropped.

File: modules/olm-updating-operator-catalog-image.adoc [hunk 2]
```diff
@@ -3,10 +3,10 @@
 // * operators/admin/olm-managing-custom-catalogs.adoc
 
 ifdef::openshift-origin[]
-:registry-image: quay.io/openshift/origin-operator-registry:4.7.0
+:registry-image: quay.io/openshift/origin-operator-registry:4.8.0
```
Comment (adellape) [inline modules/olm-updating-operator-catalog-image.adoc:6]: Same here, https://github.com/openshift/openshift-docs/pull/33720 will remove this module for 4.8, so the changes in this PR can be dropped.

**Comment by bobfuru** [comment]
LGTM!

**Pull request merged**